modules/ROOT/partials/creating-site-using-operator.adoc

```diff
@@ -0,0 +1,56 @@
+// Type: procedure
+[id='creating-site-using-operator-{context}']
+== Creating a site using the Skupper Operator
+
+
+. Create a YAML file defining the ConfigMap of the site you want to create.
++
+--
+For example, create `skupper-site.yaml`:
+
+[source,yaml,options="nowrap"]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: skupper-site
+  namespace: my-namespace
+spec:
+  channel: alpha
+  installPlanApproval: Automatic
+  name: skupper-operator
+  source: skupper-operator
+  sourceNamespace: openshift-marketplace
+  startingCSV: skupper-operator.v0.6.0
+----
```

Comment:
This is wrong. The spec above is a `Subscription` and not related to the `skupper-site` (`ConfigMap`).

Reply:
The `data` section you are mentioning below has the correct content.

Reply:
@fgiorgetti  apologies, had added spec incorrectly to try and get cluster wide instance going. I've reverted text, but note that I have gone with minimal configmap as the starting point